rollup/core/classes/RollupFieldInitializer.cls

```diff
@@ -65,8 +65,8 @@ public virtual class RollupFieldInitializer {
     return null;
   }
 
-  public SObject getSObjectFromName(String sObjectName) {
-    return ((SObject) Type.forName(sObjectName).newInstance());
+  public Schema.DescribeSObjectResult getDescribeFromName(String sObjectName) {
+    return Schema.describeSObjects(new List<String>{ sObjectName })[0];
```

Comment:
Iiiiinteresting! So this is faster than `Type.forName()`?

Reply:
I wrote a little bit about it 😅 after talking with Jeff Kranz - [here's the link](https://www.jamessimone.net/blog/joys-of-apex/idiomatic-salesforce-apex/#getting-sobjecttypes-by-name)!

Reply:
Well...... I guess it's time for me to update some code in my repos to use `Schema.describeSObjects()`!